### ISSUE TITLE
first_week_day isn't respected

### ISSUE THREAD

**Issue by stephantual** [tsileo/bakthat#30]
I'm not 100% sure about this one, however:

```
 > bakthat configure_backups_rotation
 first week day (entered 'Monday')
```

in /xxxxx/.bakthat.yml

```
      first_week_day: 0
```

Monthlies seem to default to 1st of the month (which is of course fine), but weeklies on the other hand seems to keep Saturdays rather than Mondays.

**Comment by tsileo** [comment]
I think you're right, I submitted an issue on the grandfatherson repository: https://github.com/ecometrica/grandfatherson/issues/2, I keep you updated.

Also, a little off-topic, I would love to hear your feedback on [BakManager](https://bakmanager.io) (I made a lot of updates recently), see [BakManager integration](http://docs.bakthat.io/en/latest/bakmanager_integration.html).

Using <code>BakHelper</code>, you can track a backup just by adding a <code>key</code> keyword argument when calling <code>bh.backup(key="mybackup")</code>.

Thanks!

**Comment by tsileo** [comment]
Solved with grandfatherson v1.2.
